Srcdoc

ibash opened the pull request:
Use srcdoc instead of document.open
This is to work around a crash in electron caused by document.open.

ibash committed:
Use srcdoc instead of document.open

This is to work around a crash in electron caused by document.open.

ibash committed:
2.0.2

gshahbazian commented on package.json:
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@superhuman/react-frame-component",
-  "version": "2.0.0",
```
What happened to 2.0.1? : P

ibash replied:
I don't know - someone already published it

ibash committed:
Lint

ibash committed:
2.0.3

ibash committed:
Fix bug where componentDidMount wasn't called when expected

ibash committed:
Fix all the tests

Moving to srcDoc broke all the tests since they assumed that the frame
body was available synchronously after rendering. This fixes it by
waiting for the contentDidMount/contentDidUpdate callbacks.

ibash committed:
2.0.4

ibash committed:
Remove accidental .only

ibash committed:
Fix issue where document could be null when unmounting

ibash committed:
Fix more tests

ibash committed:
2.0.5
